pkg/reconciler/knativeeventing/knativeeventing.go

```diff
@@ -282,37 +282,31 @@ func (r *Reconciler) deleteObsoleteResources(manifest *mf.Manifest, instance *ev
 		return err
 	}
 
-	// Remove the apiserversources.sources.eventing.knative.dev CRD at 0.13
-	resource.SetAPIVersion("apiextensions.k8s.io/v1beta1")
-	resource.SetKind("CustomResourceDefinition")
-	resource.SetName("apiserversources.sources.eventing.knative.dev")
-	if err := manifest.Client.Delete(resource); err != nil {
-		return err
-	}
-
-	// Remove the containersources.sources.eventing.knative.dev CRD at 0.13
-	resource.SetAPIVersion("apiextensions.k8s.io/v1beta1")
-	resource.SetKind("CustomResourceDefinition")
-	resource.SetName("containersources.sources.eventing.knative.dev")
-	if err := manifest.Client.Delete(resource); err != nil {
-		return err
+	// Remove the obsolete CRDs at 0.13
+	crds := []string{"apiserversources.sources.eventing.knative.dev",
+		"apiserversources.sources.knative.dev", "brokers.eventing.knative.dev", "channels.messaging.knative.dev",
+		"configmappropagations.configs.internal.knative.dev", "containersources.sources.eventing.knative.dev",
+		"cronjobsources.sources.eventing.knative.dev", "eventtypes.eventing.knative.dev",
+		"inmemorychannels.messaging.knative.dev", "parallels.flows.knative.dev", "pingsources.sources.knative.dev",
+		"sequences.flows.knative.dev", "sinkbindings.sources.eventing.knative.dev", "sinkbindings.sources.knative.dev",
+		"subscriptions.messaging.knative.dev", "triggers.eventing.knative.dev",
 	}
 
-	// Remove the cronjobsources.sources.eventing.knative.dev CRD at 0.13
-	resource.SetAPIVersion("apiextensions.k8s.io/v1beta1")
-	resource.SetKind("CustomResourceDefinition")
-	resource.SetName("cronjobsources.sources.eventing.knative.dev")
-	if err := manifest.Client.Delete(resource); err != nil {
-		return err
+	for _, crd := range crds {
```

Comment:
Have we considered adding a 'delete manifest' where we house cleanups until we get something like the one-off Job's proposed in the Operator re-think? It would be nice to minimize the Operator code changes, imo.